src/components/IdentityView/IdentityView.tsx

```diff
@@ -166,7 +167,10 @@ class IdentityView extends React.Component<Props, State> {
           {!(balance > 0) && (
             <button
               className="requestTokens"
-              onClick={this.openKiltFaucet}
+              onClick={this.openKiltFaucet.bind(
```

Comment:
Since you are binding it here, you can remove the bind on top.
BUT, you could also use a function, which returns another function to solve this scoping issue.

Reply:
Could you please elaborate?

Reply:
which bind on top is removable?